addons/api/addon/adapters/application.js

```diff
@@ -262,15 +266,16 @@ export default class ApplicationAdapter extends RESTAdapter.extend(
    * error format.
    *
    * @param {object} payload
+   * @param status
    * @return {object}
    */
-  transformValidationErrors(payload) {
+  transformValidationErrors(payload, status) {
     const fieldErrors = get(payload, 'details.request_fields') || [];
     // Normalize the primary error message into "base", which is an error
     // that applies to the whole model instance
     const baseError = {
       isInvalid: true,
-      status: payload.source,
+      status,
       code: payload.code,
```

Comment:
It seems like status and code are always null? We don't get back any status or code in the error payload from the backend so I'm not sure if this is a mistake, the API had changed since we implemented this, or I'm missing something. I needed the status though and piped it through regardless. 

Does anyone know about these fields?

Reply:
I'm curious- what do we get from the error payload? 
As far as I remember, we got the code and status from the error payload

Reply:
This is an example error payload we get:
```
{
  "kind": "InvalidArgument",
  "message": "Error in provided request.",
  "details": {
    "request_fields": [
      {
        "name": "name",
        "description": "This field is required."
      },
      {
        "name": "type",
        "description": "Unknown type provided."
      }
    ]
  }
}
```

I checked some different errors and none of them seemed to have a status or code in the payload

Reply:
I double checked, the API was responding with these fields in `0.1.0` but the API got updated in `0.2.0` so we haven't been passing correctly for a while now...